xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -320,6 +322,10 @@ void CDVDSubtitlesLibass::ApplyStyle(style subStyle, renderOpts opts)
       // text line to become more discontinuous (test on LibAss 15.1)
       style->Spacing = 0;
 
```

Comment:
Ups sorry, typo: paragraph. Can you change for merging?
Thanks

Reply:
yes done